source/features/faster-pr-diff-options.tsx

```diff
@@ -90,31 +83,43 @@ function init(): false | void {
 		prTitle.title = prTitle.textContent!;
 	}
 
-	// Remove previous options UI
-	const singleCommitUI = select('[data-ga-load^="Diff, view"]');
-
-	if (singleCommitUI) {
-		singleCommitUI.remove();
-		return;
-	}
+	// Only show the native dropdown on medium and small screens #2597
+	select('.js-diff-settings')!.closest('details')!.classList.add('d-lg-none');
+	// Make space for the new button by removing "Changes from" #655
+	(select('.js-commits-filtered') ?? select('[data-hotkey="c"]'))!.firstChild!.remove();
+	// Remove extraneous padding around "Clear filters" button
+	select('.subset-files-tab')?.classList.replace('px-sm-3', 'ml-2');
```

Comment:
Is `?.` still needed since we split the init?

Also there should be a space above each comment in this block to improve readability

Reply:
> Is ?. still needed since we split the init?

Yes because "Clear filters" is only there when there are some filters enabled.

> Also there should be a space above each comment in this block to improve readability

:+1: